pkgs/by-name/fy/fyi/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitea,
+  pkg-config,
+  meson,
+  ninja,
+  dbus,
+  scdoc,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "fyi";
+  version = "1.0.4";
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "dnkl";
+    repo = "fyi";
+    rev = finalAttrs.version;
+    hash = "sha256-UGkShHziREQTkQUlbFXT1144BiBApFVbCvu5A1DuoMI=";
+  };
+
+  depsBuildBuild = [ pkg-config ];
```

Comment:
Is `depsBuildBuild` necessary here? Does cross fail without it?

Reply:
Yes. Otherwise:
```
Did not find pkg-config by name 'pkg-config'
Found pkg-config: NO
Did not find CMake 'cmake'
Found CMake: NO
Build-time dependency scdoc found: NO

meson.build:19:8: ERROR: Dependency lookup for scdoc with method 'pkgconfig' failed: Pkg-config for machine build machine not found. Giving up.
```

Reply:
I mostly meant putting `pkg-config` in `nativeBuildInputs`, not removing it.